test/regression.uts

```diff
@@ -8928,4 +8928,21 @@ if WINDOWS:
     conf.route.resync()
     conf.route6.resync()
 
-True
\ No newline at end of file
+True
+
+############
+############
++ Scapy version
+
+= _version()
+
+version = scapy._version()
+import os
+assert(os.path.exists("scapy/VERSION"))
+
+import mock
+with mock.patch("scapy._version_from_git_describe") as version_mocked:
+  version_mocked.side_effect = Exception()
+  assert(scapy._version() == "2.3.3.dev696")
```

Comment:
Isn't this going to change ?

Reply:
I don't know how the mock library works, but it looks like it has been integrated as a standard library in python 3 in the unittest module. So I'm not sure 'import mock' will work

Reply:
Nah this point works :) We're manually installing the mock module as unittest.mock isn't available on python 2.7.